pkg/common/common.go

```diff
@@ -47,23 +46,6 @@ type KuadrantPolicy interface {
 	GetRulesHostnames() []string
 }
 
-func Ptr[T any](t T) *T {
-	return &t
-}
-
-// FetchEnv fetches the value of the environment variable with the specified key,
-// or returns the default value if the variable is not found or has an empty value.
-// If an error occurs during the lookup, the function returns the default value.
-// The key and default value parameters must be valid strings.
-func FetchEnv(key string, def string) string {
-	val, ok := os.LookupEnv(key)
-	if !ok {
-		return def
-	}
-
-	return val
-}
-
```

Comment:
These functions are already provided by:
- https://pkg.go.dev/k8s.io/utils@v0.0.0-20230726121419-3b25d923346b/ptr#To
- https://pkg.go.dev/k8s.io/utils@v0.0.0-20230726121419-3b25d923346b/env#GetString